spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -548,7 +1020,7 @@ def expect_try_taking_new_pictures(expected_to_be_present = true)
     end
   end
 
-  def expect_review_issues_body_message(translation_key)
+  def expect_review_body_message(translation_key)
```

Comment:
whats the intention for this renaming? the former seemed clearer to me 🤔

Reply:
lint was complaining about lines being too long, I can deal with lint another way. Resolving now...